### ISSUE TITLE
Sy select probleme code HTML invalide

### ISSUE THREAD

**Issue by verushkarossi** [assurance-maladie-digital/design-system-v3#1252]
## Type d'issue

♿ Accessibilité

### Impact

Bloquant : 

## Description du problème

- Element [input](https://html.spec.whatwg.org/multipage/#the-input-element) is missing required attribute aria-expanded
 
-Bad value for attribute autocomplete on element [input](https://html.spec.whatwg.org/multipage/#the-input-element): Must not be empty.

## Critères RGAA concernés

8.2 Pour chaque page web, le code source généré est-il valide selon le type de document spécifié (hors cas particuliers) ?

## Environnement

- Technologies d'assistance utilisées : 
- Navigateur : 
- OS :
